utils/_features.py

```diff
@@ -2218,14 +2218,30 @@ def iast_source_sql(test_object):
         pytest.mark.features(feature_id=344)(test_object)
         return test_object
 
-    @staticmethod
-    def tracing_configuration_consistency(test_object):
-        """Enforces standardized behaviors for configurations across the tracing libraries.
-
-        https://feature-parity.us1.prod.dog/#/?feature=325
-        """
-        pytest.mark.features(feature_id=325)(test_object)
-        return test_object
+    # @staticmethod
+    # def tracing_configuration_consistency(test_object):
+    #     """Enforces standardized behaviors for configurations across the tracing libraries.
+
+    #     https://feature-parity.us1.prod.dog/#/?feature=325
+    #     """
+    #     pytest.mark.features(feature_id=325)(test_object)
+    #     return test_object
+    ### Replace config consistency with the following features
+    # @features.tracing_enablement
+    # @features.ddtrace_log_directory
```

Comment:
```suggestion
    # @features.trace_log_directory
```
to match the updated name above.